### PR TITLE
add: `ticktick-deb`

### PR DESCRIPTION
**Pull request by Arrowsome**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packages/ticktick-deb/ticktick-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    url="https://ticktick-download-linux.s3.amazonaws.com/download/linux/linux_deb_x64/ticktick-${version}-arm64.deb"
+    hash="66f19c06aedfcaefece90d09db23267872e4bd153f067ad908106a9d306ee014"
+    arch=("arm64")
+    ;;
+  *)
+    return 1
+    ;;
+esac
+homepage="https://ticktick.com"
+description="A powerful to-do & task management app with seamless cloud synchronization across all your devices. Whether you need to schedule an agenda, make memos, share shopping lists, collaborate in a team, or even develop a new habit, TickTick is always here to help you get stuff done and keep life on track"
```
Comment (Henryws) [inline packages/ticktick-deb/ticktick-deb.pacscript:23]: This description is too long.

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `repology` array if you can.

File: packages/ticktick-deb/ticktick-deb.pacscript (hunk 1 above)
Comment (Arrowsome) [reply packages/ticktick-deb/ticktick-deb.pacscript]: Initially, my understanding was a `.pacscript` description should be the same as the `.deb` file's. I made it shorter in the new commit, but I can trim it down even more if necessary.

Comment (Henryws) [reply packages/ticktick-deb/ticktick-deb.pacscript]: It should be, but the description in the `.deb` is *wayyyy* to long for my liking.

**Review by Henryws — APPROVED** [review]

**Comment by Henryws** [comment]
@Arrowsome, create a new PR to add `maintainer` to this pacscript. Sorry I forgot, it's 7AM for me 👀

**Pull request merged**